### PR TITLE
Downgrade TelepathyGLib

### PR DESCRIPTION
**Pull request by sparshg**

### CODE REVIEW

File: org.sugarlabs.BaseApp.json [hunk 1]
```diff
@@ -174,13 +174,9 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://github.com/TelepathyIM/telepathy-glib.git",
-                    "tag": "telepathy-glib-0.99.11",
-                    "commit": "d00ccc12e7c27b42538389be4595f658ac2cc4eb",
-                    "x-checker-data": {
```
Comment (tchx84) [inline org.sugarlabs.BaseApp.json:180]: we should keep x-checker-data, by updating the pattern to look for 0.24.x

**Review by tchx84 — CHANGES_REQUESTED** [review]

**Review by tchx84 — CHANGES_REQUESTED** [review]
Also, please check the git log and stick to the same convention for commit messages.

File: org.sugarlabs.BaseApp.json (hunk 1 above)
Comment (sparshg) [reply org.sugarlabs.BaseApp.json:180]: So the pattern should be `^telepathy-glib-0.24.\\d$` if I am correct?

Comment (tchx84) [reply org.sugarlabs.BaseApp.json:180]: Test it with https://regex101.com/ and test it with external-data-checker tool (see https://github.com/flathub/org.sugarlabs.BaseApp/tree/branch/23.06#how-to-update)

**Comment by sparshg** [comment]
Tested with checker tool, commit message changed.

File: org.sugarlabs.BaseApp.json [hunk 2]
```diff
@@ -174,12 +174,12 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://github.com/TelepathyIM/telepathy-glib.git",
-                    "tag": "telepathy-glib-0.99.11",
-                    "commit": "d00ccc12e7c27b42538389be4595f658ac2cc4eb",
+                    "url": "https://www.github.com/TelepathyIM/telepathy-glib.git",
```
Comment (tchx84) [inline org.sugarlabs.BaseApp.json]: are you seeing a re-direction warning? I don't think we need the `www` part.

**Review by tchx84 — CHANGES_REQUESTED** [review]

**Review by tchx84 — CHANGES_REQUESTED** [review]
check all comments

**Review by tchx84 — APPROVED** [review]

**Pull request merged**